qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -81,8 +81,10 @@ def run(self, qobj: Qobj, job_name: Optional[str] = None) -> IBMQJob:
             IBMQJob: an instance derived from BaseJob
         """
         # pylint: disable=arguments-differ
-        kwargs = {'use_object_storage': True,
+        use_object_storage = self.configuration().allow_object_storage
+        kwargs = {'use_object_storage': use_object_storage,
```

Comment:
We can simplify things a bit after the deprecation of `v1` - instead of using the temporary variables, can you simply pass `use_object_storage` to the constructor (and just drop `use_websockets`, to make it rely on the default value)?

```
job = IBMQJob(self, None, self._api, qobj=qobj, use_object_storage=self.configuration....)
```

Reply:
Yeah, simplifying would be a good idea, will do! 👍